predicators/envs/burger.py

```diff
@@ -176,14 +176,24 @@ def _get_tasks(self, num: int,
         hidden_state[bottom_bun] = {"is_held": 0.0}
 
         goal = {
```

Comment:
I've commented the real goal out because I'm never using the full goal during testing -- it's slower to plan to it and more prone to errors. Rather than just completely replace it for the next `N` PRs, I say we keep it commented out here. Since no one else is working on BurgerEnv except us, I think it's fine to leave it in this informal state for now.